### PR TITLE
feature/new integration: Onboarding rockerbox for v1.1

### PR DESCRIPTION
**Pull request by yashasvibajpai**
## Description of the change

> Onboarding Rockerbox

## Type of change
- [ ] Bug fix (non-breaking change that fixes an issue)
- [x] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Related issues

> Fix [#1]() 

### Development

- [ ] Lint rules pass locally
- [ ] The code changed/added as part of this pull request has been covered with tests
- [ ] All tests related to the changed code pass in development

### Code review 

- [ ]  This pull request has a descriptive title and information useful to a reviewer. There may be a screenshot or screencast attached
- [ ] "Ready for review" label attached to the PR and reviewers mentioned in a comment
- [ ] Changes have been reviewed by at least one other engineer
- [ ] Issue from task tracker has a link to this pull request

- - -
This change is [Reviewable](https://reviewable.io/reviews/rudderlabs/rudder-sdk-js/609)

### CODE REVIEW

File: integrations/Rockerbox/index.js [hunk 1]
```diff
@@ -1,3 +1,3 @@
-import Rockerbox from './browser';
+import Rockerbox from "./browser";
```
Comment (saikumarrs) [inline integrations/Rockerbox/index.js:1]:
```suggestion
import Rockerbox from './browser';
```
The formatter should've taken care of this during the commit. Did you skip the checks?

Comment (MoumitaM) [inline integrations/Rockerbox/index.js:1]: Use single quote `"./browser"`.

**Review by MoumitaM — APPROVED** [review]

File: integrations/Rockerbox/index.js (hunk 1 above)
Comment (yashasvibajpai) [reply integrations/Rockerbox/index.js:1]: Thanks you, fixed

Comment (yashasvibajpai) [reply integrations/Rockerbox/index.js]: Yes, by mistake, ran it with -n flag though I did delibrately change `singlequote` to `double` as I could see from the last mouse-flow integration PR.
Fixed this.

**Review by saikumarrs — APPROVED** [review]

**Review by utsabc — APPROVED** [review]

**Pull request merged**